packages/@aws-cdk/aws-apprunner-alpha/lib/service.ts

```diff
@@ -1140,6 +1140,17 @@ export class Service extends cdk.Resource {
     this.secrets.push({ name: name, value: secret.arn });
   }
 
+  /**
+   * This method exposes the Instance Role after creating it if not set.
+   * @returns iam.IRole
+   */
+  public obtainInstanceRole(): iam.IRole {
```

Comment:
What if we instead made `Service` implement `IGrantable`? Then we could just do
things like

```ts
bucket.grantRead(service);
```